tests/test_quotes.py

```diff
@@ -0,0 +1,35 @@
+from datetime import date
+from pathlib import Path
+
+import pandas as pd
+import yfinance
+from pytest import approx
+
+from investd.config import PERSIST_PATH
+from investd.quotes import adjust_symbol, generate_quotes_csv, load_quotes
+
+
+def test_adjust_symbol():
+    assert adjust_symbol("AAPL") == "AAPL"
+    assert adjust_symbol("BETASPXP.PL") == "BETASPXP.WA"
+    assert adjust_symbol("INR.FR") == "INR.PA"
+    assert adjust_symbol("CSPX.UK") == "CSPX.L"
+
+
+def test_generate_quotes_csv(monkeypatch, df_quotes: pd.DataFrame):
```

Comment:
This test now only checks that generate_quotes_csv saves a csv returned from yfinance call. Maybe you could also assert the parameters, with which yfinance was called? Because now, most of the logic from _generate_quotes_csv_ is not tested (e.g., loading txs, checking min date, adjusting symbols etc)

Reply:
alternatively, you could introduce 3 different methods to prepare a call to yfinance, execute it, and save the results to file. Then, you could separately test "call preparation" logic without mocking anything. Saving CSV to disk probably doesn't need testing, same as calling yfinance, unless you want to make it an integration test and actually call yfinance.

Reply:
That's true! I completely missed that this test wasn't testing much in the end.

I added an assertion to check if the parameters have been correctly provided to the monkey patch.